### DISCUSSION TITLE
Post your Farming rigs

### DISCUSSION THREAD

**Discussion by hoffmang9** [Chia-Network/chia-blockchain#507] [Show and tell]
Let's see photos of your farming set ups. Some description and components used would be nice too.

**Comment by dorkmo** [comment]
![KIMG1353](https://user-images.githubusercontent.com/1923070/98062145-0c081400-1e13-11eb-90e9-3cdcc328631a.JPG)

some wd elements, a startech 10 port hub (hb30a10ame) and two power strips (hubbell pt153112p)

humble crypto farm

**Reply by dorkmo** [reply]
i just got a Rasberry Pi 400 desktop setup with chia gui using the ubuntu distribution https://ubuntu.com/raspberry-pi . running great so far. had to install some of the dependencies listed on https://github.com/Chia-Network/chia-blockchain/wiki/Raspberry-Pi

**Reply by andre-bender** [reply]
Why is everyone using those cheap hdds? They are not meant to run 24/7. Is there already some info about the lifetime on hdds just on read-only?

**Reply by dorkmo** [reply]
Hi hi, I set this up before i knew anything about hard drives :D please forgive my noob setup!

**Reply by 4jin** [reply]
> 我刚刚使用ubuntu发行版https://ubuntu.com/raspberry-pi获得了带有chia gui的Rasberry Pi 400桌面设置。到目前为止运行良好。必须安装[https://github.com/Chia-Network/chia-blockchain/wiki/Raspberry-Pi上](https://github.com/Chia-Network/chia-blockchain/wiki/Raspberry-Pi)列出的一些依赖项[](https://github.com/Chia-Network/chia-blockchain/wiki/Raspberry-Pi)

Can you share some tutorials, looking for a long time did not find the relevant information

**Reply by dorkmo** [reply]
@4jin we can discuss here: https://github.com/Chia-Network/chia-blockchain/discussions/1678 or https://github.com/Chia-Network/chia-blockchain/discussions/1739

**Reply by giovadaal** [reply]
hi men, can you help me to install firmware chia on rasberry pi 4?

**Comment by Lenkus1** [comment]
Pi4 with WD elements and Seagate expansion desktops on a 7 port tp-link hub. DIY cupboard with lots of room for more ;) Aiming for 100TB at launch.
![IMG_20201106_081041](https://user-images.githubusercontent.com/58724760/98337138-6e763580-2008-11eb-879c-b1f1cd929635.jpg)

**Reply by dorkmo** [reply]
lots of white! needs more flair!

**Reply by IT-Maxy** [reply]
> Pi4 with WD elements and Seagate expansion desktops on a 7 port tp-link hub. DIY cupboard with lots of room for more ;) Aiming for 100TB at launch.
> ![IMG_20201106_081041](https://user-images.githubusercontent.com/58724760/98337138-6e763580-2008-11eb-879c-b1f1cd929635.jpg)

how you managed to connect your plotter to the raspberry? im struggling to make it :(

**Comment by xorinox** [comment]


**Reply by Lenkus1** [reply]
Very nice!

**Reply by dorkmo** [reply]
@Lenkus1 we've been out done by the 16 port hub! 🤣

**Reply by Lenkus1** [reply]
looking good

**Reply by dorkmo** [reply]
are you planning on adding more levels to your rig? im starting to image your future warehouse full of these from floor to ceiling lol

**Reply by HoboDiem** [reply]
What Brand HDD are those? Happy with them or you you revise now that it's up and running?

**Comment by Teledyol** [comment]
Mac Pro • 3.5 GHz 6-Core Intel Xeon E5 • 128 GB 1066 MHz DDR3 • AMD FirePro D300 2 GB • SSD: 2T Aura Pro X2 • Farm Disk: 32T OWC Mercury - RAID 0

![Mar_24_Chia](https://user-images.githubusercontent.com/3718370/112363432-b97ca680-8c92-11eb-9427-c35925b04bde.png)

Drive Icon is Lorine Chia, of course…

![Lorine_Chia](https://user-images.githubusercontent.com/3718370/112364175-7c64e400-8c93-11eb-8945-87b92ac55fdb.png)

**Comment by mgxmurat** [comment]
greetings,  My humble little 12+12+10+10+14 tb capacity chia farm.. i7 10700F 8X8 GB=32GB RAM  1X120GB SSD HDD Win10 Pro 1x1TB sam.980 pro M2ssd  1x2Tb adata M2 ssd  

![IMG_20210626_024533](https://user-images.githubusercontent.com/84076940/123495406-3cafee80-d62c-11eb-9238-397b28ce4750.jpg)
![Uploading IMG_20210626_024515.jpg…]() 
![IMG_20210626_024529](https://user-images.githubusercontent.com/84076940/123495400-36ba0d80-d62c-11eb-9134-02687ff1e551.jpg)

![IMG_20210626_024525](https://user-images.githubusercontent.com/84076940/123495392-31f55980-d62c-11eb-84a4-ffdbd82d78af.jpg)
![IMG_20210626_024503](https://user-images.githubusercontent.com/84076940/123495368-1ee28980-d62c-11eb-8a74-e7798c71cb2b.jpg)
![IMG_20210626_024507](https://user-images.githubusercontent.com/84076940/123495374-25710100-d62c-11eb-870b-6345d45dcf52.jpg)
![IMG_20210626_024511](https://user-images.githubusercontent.com/84076940/123495376-299d1e80-d62c-11eb-90b1-362290b9852d.jpg)
